llama-index-core/pyproject.toml

```diff
@@ -83,7 +83,7 @@ dirtyjson = "^1.0.8"
 tqdm = "^4.66.1"
 pillow = ">=9.0.0"
 PyYAML = ">=6.0.1"
-llamaindex-py-client = "^0.1.13"
+llamaindex-py-client = "^0.1.14a1"
```

Comment:
discussed offline would prefer to not bump versions in this PR just for testing, instead you can try pip install from a branch directly: https://stackoverflow.com/questions/20101834/pip-install-from-git-repo-branch

Reply:
Yup, I'll let Logan do it.